.cirrus.yml

```diff
@@ -0,0 +1,72 @@
+check_android_task:
+  name: Run Android instrumented tests
+  env:
+    SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspressample/files/screenshots
+    TUTORIAL_SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspresso.tutorial/files/screenshots
+    ALLURE_SAMPLE_SCREENSHOTS_ROOT_DIR: /data/user/0/com.kaspersky.kaspresso.alluresupport.sample/files/screenshots
+    DISPLAY: :99
+  only_if: $CIRRUS_PR != '' && $CIRRUS_BASE_BRANCH == 'master'
+
+  container:
+    image: cirrusci/android-sdk:30
+    kvm: true
+    cpu: 8
+    memory: 24G
+  install_de_script: |
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update
+    apt-get install xfce4 -y
+    apt-get install xfce4-goodies -y
+    apt-get purge -y pm-utils xscreensaver*
+    apt-get install xvfb -y
+  start_de_background_script: |
+    Xvfb :99 -screen 0 1000x1000x16 &
+    sleep 5
+    startxfce4
+  accept_licenses_script:
+    echo yes | sdkmanager --licenses
+  install_emulator_script:
+    sdkmanager --install "system-images;android-30;google_apis;x86"
+  create_avd_script:
+    echo no | avdmanager create avd --force
+    -n test
+    -k "system-images;android-30;google_apis;x86"
+  start_avd_background_script:
+    $ANDROID_HOME/emulator/emulator
+    -avd test
+    -no-audio
+    -no-boot-anim
+    -gpu swiftshader_indirect
+    -no-snapshot
```

Comment:
where is `-no-window`? As I know, it's required for headless mode.

Reply:
I removed -no-window intentionally. It sounds strange, but issue with failing GPS test I mentioned in the comment earlier was related to emulator being started in headless mode. I found it accidentally. You could try it locally, it's 100% reproduceable, at least with API 30 AVD